uli_init/simulate.py

```diff
@@ -687,12 +700,32 @@ def build_molecule(molecule, length, para_weight):
         Use the molecule name as seen in the .json file without including .json
     length : int
         The number of monomer units in the final polymer molecule
+    sequence : str, required
+        The monomer sequence to be used when building a polymer.
+        Example) "AB" or "AAB"
+        If you want a sequence to be generated randomly, use sequence="random"
+
     para_weight : float, limited to values between 0 and 1
         The relative amount of para configurations compared to meta.
         Passed into random_sequence() to determine the monomer sequence of the
         polymer.
         A 70/30 para to meta system would require a para_weight = 0.70
 
+    Notes:
+    ------
+    Any values entered for length and sequence should be compatible.
+    This is designed to follow the sequence until it has reached it's
+    terminal length.
+
+        For example:
```

Comment:
```suggestion
    terminal length. For example::
```
This will render better in sphinx (I know this repo isn't using that yet, but you never know ;) )

Reply:
Hopefully one day :)